app/src/main/java/com/myvetpath/myvetpath/ViewSubsActivity.java

```diff
@@ -0,0 +1,168 @@
+package com.myvetpath.myvetpath;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.design.widget.Snackbar;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.support.v7.widget.Toolbar;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+public class ViewSubsActivity extends AppCompatActivity {
+
+    Intent create_sub_activity;
+    Intent sub_details_activity;
+
+    private RecyclerView mRecyclerView;
+    private RecyclerView.Adapter mAdapter;
+    private RecyclerView.LayoutManager mLayoutManager;
+
+    private String[] subsTitles;
+    private String[] subsDates;
+    private String[] subsCaseID;
+
+    public class SubsAdapter extends RecyclerView.Adapter<SubsAdapter.MyViewHolder> {
+        private String[] mDataset;
```

Comment:
This is a nitpick, but it might be easier to read these variables if "set" was capitalized. Just so we're on the same page, what exactly is "mDataset"? How are these three string arrays different from the ones declared in lines 27-29?

Reply:
Those arrays are variables to hold the data passed to the adapter. That will all change when we connect it to the database, right now they are just string arrays for the example.